app/views/users/registrations/update.js.erb

```diff
@@ -0,0 +1 @@
+location.reload();
```

Comment:
What is this for?

Reply:
Once the form (GDPR checkbox) gets submitted via AJAX request; the page needs to reload inorder to let user access the platform.